src/lib/ical/utils.test.ts

```diff
@@ -69,3 +76,32 @@ test('wraps long text into lines broken at column 100 3 times', () => {
   assert(lines.length === 4, 'wraps text into 3 times, making 4 lines');
   assert(lines.at(-1).length === 1, 'where the last line is a single asterisk');
 });
+
+test('replaces HTML with external links', () => {
+  const actual = replaceHtmlWithExternalLinks(`<a href="https://example.com/">
+  <a>
+  <a title="example" href="https://example.com/">
+  <a title="example" target="_self" href="https://example.com/">
+  <a title="example" target="_blank" href="https://example.com/">
+  example`);
+  expect(actual).toBe(`<a href="https://example.com/" target="_blank">
+  <a>
+  <a title="example" href="https://example.com/" target="_blank">
+  <a title="example" href="https://example.com/" target="_blank">
+  <a title="example" target="_blank" href="https://example.com/">
+  example`);
+});
+
+test('additional replaces HTML with external links tests', () => {
+  const actual =
+    replaceHtmlWithExternalLinks(`<a title="example" target="_self"  href="https://example.com/">
+  <a title="example" target="_self"href="https://example.com/">
+  <a target="_self" title="example" href="https://example.com/">
+  <a title="example" target="_self" target="poggers" target="not poggers"  href="https://example.com/">
+  example`);
```

Comment:
What if there are any tags that begin with `<a`? For example, does the input `<article href="https://example.com/">example</article>` remain unchanged as expected?